### ISSUE TITLE
[Q] Auto setup an app in Service Commander?

### ISSUE THREAD

**Issue by ravisankar-PIO** [ThePrez/ServiceCommander-IBMi#227]
Hi,
     **A)** Is it possible to setup an application (let's say Jenkins) to start via SC by passing all the answers to the interactive questions as parameters? E.g., 

```bash
scinit java -jar ~/jenkins.war --httpPort=9095 <followed by parameters for interactive questions???>
```
**B)** If the above method is not possible, then can I create a YAML file manually (of course with correct format) and place it in the ~/.sc/services/ folder such as `~/.sc/services/jenkins.yml` and then issue a `sc start jenkins`?

In other words, **What is the ideal way of automating the SCINIT**?

Thank you,
Ravi.

**Comment by chrjorgensen** [comment]
@ravisankar-PIO The [`scinit`](https://github.com/ThePrez/ServiceCommander-IBMi#using-the-scinit-tool) tool is an interactive proces and not suited for automation.

Instead you should [create/build the YAML file](https://github.com/ThePrez/ServiceCommander-IBMi?tab=readme-ov-file#directly-creatingediting-yaml-configuration-files) and store it in `~/.sc/services` (for user services) or in `/QOpenSys/etc/sc/services` for common services. Then you can run `sc start <service>`.

HTH.
